test/testfolderstatusmodel.cpp

```diff
@@ -29,7 +30,7 @@ using namespace OCC::Utility;
 class TestFolderStatusModel : public QObject
 {
     Q_OBJECT
-
+    static inline constexpr auto _modelTesterMode = QAbstractItemModelTester::FailureReportingMode::QtTest;
```

Comment:
nitpick: there is no point in making this variable `inline`, we are not using it in any other translation units. Since this class is not being used anywhere else we also don't need to make this `static`

```suggestion
    constexpr auto _modelTesterMode = QAbstractItemModelTester::FailureReportingMode::QtTest;
```